tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -144,5 +221,35 @@ private static string GetManifestRegistry()
             JObject manifest = JObject.Parse(manifestJson);
             return (string)manifest["registry"];
         }
+        private void VerifyHttpResponseFromContainer(string containerName, string urlPath)
+        {
+            var retries = 30;
+
+            // Can't use localhost when running inside containers or Windows.
+            var url = $"http://{DockerHelper.GetContainerAddress(containerName)}" + urlPath;
+
+            using (HttpClient client = new HttpClient())
+            {
+                while (retries > 0)
+                {
+                    retries--;
+                    System.Threading.Thread.Sleep(TimeSpan.FromSeconds(2));
+                    try
+                    {
+                        using (HttpResponseMessage result = client.GetAsync(url).Result)
+                        {
+                            result.EnsureSuccessStatusCode();
+                        }
+                        return;
+                    }
+                    catch (Exception ex)
+                    {
+                        DockerHelper.OutputHelper.WriteLine($"Request to {url} failed - retrying: {ex.ToString()}");
```

Comment:
I was debating about this, not sure which way you want this.

Reply:
Because exposing OutputHelper on DockerHelper is leaking implementation details.  The purpose of DockerHelper is to provide the APIs for working with Docker.  OutputHelper is not related to Docker.  Since this test class is passed the OutputHelper in the ctor, it should hold onto a reference for use within the test methods.